DigitalLearningSolutions.Web/ViewModels/TrackingSystem/DelegateApprovalsViewModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem
+{
+    using System;
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Web.ViewModels.Common;
+
+    public class DelegateApprovalsViewModel
+    {
+        public DelegateApprovalsViewModel(IEnumerable<ApprovableDelegate> delegates)
+        {
+            Delegates = delegates;
+        }
+
+        public IEnumerable<ApprovableDelegate> Delegates { get; set; }
+    }
+
+    public class ApprovableDelegate
+    {
+        public int Id { get; set; }
+        public string FirstName { get; set; }

```

Comment:
I wonder if we should have FullName field here and we set it in the constructor later (to move the logic from HTML to C# code)